src/Workleap.Extensions.Configuration.Secrets/TokenCredentialProvider.cs

```diff
@@ -21,13 +22,22 @@ public sealed class TokenCredentialProvider : ITokenCredentialProvider
         "Local", "LocalDocker", "Test", "Tests", "CI", Environments.Development, "DevelopmentDocker",
     };
 
+    private static readonly string[] KnownContinuousIntegrationEnvironmentVariables = {
```

Comment:
Could we set the same source as inspiration as the xunit project: https://github.com/workleap/wl-extensions-xunit/blob/bd25cb5d86f04815616c4d8e9bf16d4f6b0de7be/src/Workleap.Extensions.Xunit/BaseIntegrationFixture.cs#L78